docs/rpc/http/getTokenAccountsByOwner.mdx

```diff
@@ -108,6 +102,9 @@ that of the [Token Balances Structure](/docs/rpc/json-structures#token-balances)
 can be expected inside the structure, both for the `tokenAmount` and the
 `delegatedAmount` - with the latter being an optional object.
 
+> Note: The second parameter must include either `mint` or `programId`. You
```

Comment:
```suggestion
> Note: The second parameter must be either `mint` or `programId`. You
```

Reply:
@TanmayDhobale Please don't close discussions i you don't want to implement them. Instead say why you don't like the suggestion.